clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -284,6 +284,12 @@ Changes in existing checks
   <clang-tidy/checks/performance/faster-string-find>` check to properly escape
   single quotes.
 
+- Improved :doc:`performance-noexcept-move-constructor
+  <clang-tidy/checks/performance/noexcept-move-constructor>` and
+  :doc:`performance-noexcept-swap<clang-tidy/checks/performance/noexcept-swap>`
```

Comment:
duplicate this change note, here leave one for move-constructor, and part about swap merge with next entry in line 293. So we could have 1 entry per check.